src/utils/conversation.js

```diff
@@ -60,5 +91,6 @@ export function shouldIncludeArchived(conversation, showArchived) {
 export function filterConversation(conversation, filters) {
 	return filters.length === 0
 		|| ((!filters.includes('unread') || hasUnreadMessages(conversation))
-		&& (!filters.includes('mentions') || hasUnreadMentions(conversation)))
+		&& (!filters.includes('mentions') || hasUnreadMentions(conversation))
+		&& ((!filters.includes('events') && shouldIncludeEvents(conversation)) || isEvent(conversation)))
```

Comment:
Now it shows all events 👀. Please check if that works correctly for you:
```js
	return filters.length === 0
		? shouldIncludeEvents(conversation)
		: ((!filters.includes('unread') || hasUnreadMessages(conversation))
			&& (!filters.includes('mentions') || hasUnreadMentions(conversation))
			&& (!filters.includes('events') ? shouldIncludeEvents(conversation) : isEvent(conversation)))
```
As I got it, events should be filtered with empty filters as well